share/static/html/page.html

```diff
@@ -107,7 +107,7 @@ <h1 class="mdl-typography--display-2">
   {% block footer %}
   <footer class="mdl-mini-footer everware-footer">
       <div class="mdl-mini-footer__left-section">
-        Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
+        v0.5.0 Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
```

Comment:
I tried to do that by the jinja's templates , but there are some problems. The footer is in the page.html , that is extentened by others html pages. 
1)One page is rendered in the jupyterhub library, its "spawn.html".
2) For every render of each pages we have to add "__version__" , but i thought its easier just to modify one line in page.html